examples/CMakeLists.txt

```diff
@@ -1,7 +1,11 @@
+ADD_CUSTOM_TARGET(checkExamples
+        COMMAND ctest -C checkExamples -j 4)
```

Comment:
Any reason to not make this the same as https://github.com/AutoPas/AutoPas/pull/55/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R203?

Reply:
jup, on normal pc's you don't have 8 procs